tests/components/panel_iframe/test_init.py

```diff
@@ -1,100 +1,87 @@
 """The tests for the panel_iframe component."""
-import unittest
+import pytest
 
-from homeassistant import setup
 from homeassistant.components import frontend
+from homeassistant.setup import async_setup_component
 
-from tests.async_mock import patch
-from tests.common import get_test_home_assistant
 
+@pytest.mark.parametrize(
+    "config_to_try",
+    (
+        {"invalid space": {"url": "https://home-assistant.io"}},
+        {"router": {"url": "not-a-url"}},
+    ),
+)
+async def test_wrong_config(hass, config_to_try):
+    """Test setup with wrong configuration."""
+    assert not await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)